metagraph/src/cli/config/config.cpp

```diff
@@ -1241,16 +1243,18 @@ if (advanced) {
 if (advanced) {
             fprintf(stderr, "\t   --sparse \t\tuse row-major sparse matrix for row annotation [off]\n");
 }
+            fprintf(stderr, "\t   --json \t\toutput query results in JSON format [off]\n");
             fprintf(stderr, "\n");
             fprintf(stderr, "\t   --count-labels \t\tcount labels for k-mers from querying sequences [off]\n");
             fprintf(stderr, "\t   --count-kmers \t\tweight k-mers with their annotated counts (requires count annotation) [off]\n");
             fprintf(stderr, "\t   --count-quantiles [FLOAT ...] \tk-mer count quantiles to compute for each label [off]\n"
                             "\t                                 \t\tExample: --count-quantiles '0 0.33 0.5 0.66 1'\n"
                             "\t                                 \t\t(0 corresponds to MIN, 1 corresponds to MAX)\n");
+            fprintf(stderr, "\t   --query-counts \t\tquery k-mer counts (requires count or coord annotation) [off]\n");
             fprintf(stderr, "\t   --query-coords \t\tquery k-mer coordinates (requires coord annotation) [off]\n");
-            fprintf(stderr, "\t   --verbose-coords \t\tdo not collapse continuous matched coord ranges in result (for query-coords) [off]");
+            fprintf(stderr, "\t   --verbose-output \t\tdo not collapse continuous coord or count ranges (for query-coords and query-counts) [off]\n");
```

Comment:
```suggestion
            fprintf(stderr, "\t   --verbose-output \t\tdo not collapse contiguous coord or count ranges (for query-coords and query-counts) [off]\n");
```

Reply:
Isn't 'continuous' more appropriate in the context of counts?

Reply:
"Contiguous" would be more correct for coordinates, but I agree that it doesn't work for counts. For counts, we could say "runs" to refer to ranges of matching values.

Reply:
I think it's better to deliver the intuition right than to be extremely accurate.

Among these two, continuous seems to be a better option to me.